tests/components/analytics/doughnut_chart.test.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+
+import {shallow, mount} from 'enzyme';
+
+import DoughnutChart from 'components/analytics/doughnut_chart.jsx';
+
+jest.mock('chart.js');
+
+describe('components/analytics/doughnut_chart.jsx', () => {
+    test('should match snapshot, on loading', () => {
+        const wrapper = shallow(
+            <DoughnutChart
+                title='Test'
+                height={400}
+                width={600}
+            />
+        );
+
+        expect(wrapper).toMatchSnapshot();
+    });
+
+    test('should match snapshot, loaded without data', () => {
+        const data = {};
+
+        const wrapper = shallow(
```

Comment:
Should probably use `mountWithIntl` instead of `shallow` as I couldn't see the difference in snapshots between "loaded without data" and "loaded with data".